frontend/dev-proxy/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "dev-proxy",
+  "version": "1.0.0",
+  "description": "",
+  "main": "build/dist.js",
+  "scripts": {
+    "dev": "NODE_ENV=development nodemon build/dist.js",
+    "serve": "NODE_ENV=development webpack",
```

Comment:
This doesn't work cross-environment. Can we put the `NODE_ENV=development` declaration in the `.env` instead? Seems to work like that on my end.

Reply:
Basically that is what I wanted to confirm. This was slightly different since it didn't use export but let me take care of it :)

Reply:
![image](https://user-images.githubusercontent.com/27592/83359437-8a4f1e80-a37a-11ea-9632-fda50075a8ce.png)
That's all I did.

1 other thing is somehow we need a script in the frontend/app package.json to use the `.env.development.local` config instead of the default `.env`. I'm not sure how to do that (I had to change the port in the .env file to get it to work).

Reply:
We shouldn't need it. I checked locally and anything in the `.env.development.local` was overriding `.env` for the app properly.

Reply:
How does it know to do that?

Reply:
There are some priority rules on how these files are loaded. I had to try a few things before figuring out the exact file but this seemed to work without any need for modifications.

Reply:
I'm getting an error doing `npm run serve` on b6ce57c 

```
❯(rotkehlchen)  npm run serve

> dev-proxy@1.0.0 serve C:\Users\Gebruiker\Source\Repos\rotki\frontend\dev-proxy
> webpack

C:\Users\Gebruiker\Source\Repos\rotki\frontend\dev-proxy\node_modules\webpack-cli\bin\cli.js:93
                                throw err;
                                ^

TypeError: Cannot convert undefined or null to object
    at Function.keys (<anonymous>)
    at Object.<anonymous> (C:\Users\Gebruiker\Source\Repos\rotki\frontend\dev-proxy\webpack.config.js:13:8)
    at Module._compile (C:\Users\Gebruiker\Source\Repos\rotki\frontend\dev-proxy\node_modules\v8-compile-cache\v8-compile-cache.js:192:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1167:10)
    at Module.load (internal/modules/cjs/loader.js:996:32)
    at Function.Module._load (internal/modules/cjs/loader.js:896:14)
    at Module.require (internal/modules/cjs/loader.js:1036:19)
    at require (C:\Users\Gebruiker\Source\Repos\rotki\frontend\dev-proxy\node_modules\v8-compile-cache\v8-compile-cache.js:161:20)
    at WEBPACK_OPTIONS (C:\Users\Gebruiker\Source\Repos\rotki\frontend\dev-proxy\node_modules\webpack-cli\bin\utils\convert-argv.js:114:13)
    at requireConfig (C:\Users\Gebruiker\Source\Repos\rotki\frontend\dev-proxy\node_modules\webpack-cli\bin\utils\convert-argv.js:116:6)
    at C:\Users\Gebruiker\Source\Repos\rotki\frontend\dev-proxy\node_modules\webpack-cli\bin\utils\convert-argv.js:123:17
    at Array.forEach (<anonymous>)
    at module.exports (C:\Users\Gebruiker\Source\Repos\rotki\frontend\dev-proxy\node_modules\webpack-cli\bin\utils\convert-argv.js:121:15)
    at C:\Users\Gebruiker\Source\Repos\rotki\frontend\dev-proxy\node_modules\webpack-cli\bin\cli.js:71:45
    at Object.parse (C:\Users\Gebruiker\Source\Repos\rotki\frontend\dev-proxy\node_modules\webpack-cli\node_modules\yargs\yargs.js:567:18)
    at C:\Users\Gebruiker\Source\Repos\rotki\frontend\dev-proxy\node_modules\webpack-cli\bin\cli.js:49:8
    at Object.<anonymous> (C:\Users\Gebruiker\Source\Repos\rotki\frontend\dev-proxy\node_modules\webpack-cli\bin\cli.js:366:3)
    at Module._compile (internal/modules/cjs/loader.js:1147:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1167:10)
    at Module.load (internal/modules/cjs/loader.js:996:32)
    at Function.Module._load (internal/modules/cjs/loader.js:896:14)
    at Module.require (internal/modules/cjs/loader.js:1036:19)
    at require (internal/modules/cjs/helpers.js:72:18)
    at Object.<anonymous> (C:\Users\Gebruiker\Source\Repos\rotki\frontend\dev-proxy\node_modules\webpack\bin\webpack.js:156:2)
    at Module._compile (internal/modules/cjs/loader.js:1147:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1167:10)
    at Module.load (internal/modules/cjs/loader.js:996:32)
    at Function.Module._load (internal/modules/cjs/loader.js:896:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:71:12)
    at internal/main/run_main_module.js:17:47
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! dev-proxy@1.0.0 serve: `webpack`
npm ERR! Exit status 1
```